src/File.cpp

```diff
@@ -1,8 +1,17 @@
 #include "File.hpp"
 
+#include <fcntl.h>
 #include <sys/stat.h>
+#include <unistd.h>
```

Comment:
```suggestion
#include <cunistd>
```
이런게 가능한 헤더들은 추후 확인해서 한 번에 정리하시죠!